packages/sampler/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The following is a curated list of changes in the Enact Sampler, newest changes
 
 No significant changes.
 
+## [5.0.0-alpha.1] - 2024-07-11
+
+No significant changes.
```

Comment:
I guess we missed the change log for this.
Could you add below?
```suggestion
### Changed

- `sampler` look and feel by updating to Storybook 8
```

Reply:
ok